week-6/Homework/mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,70 @@
 // Write your code here
+const autoPreviusButton = document.getElementById("auto-backward");
+const previusButton = document.getElementById("backward");
+const pauseButton = document.getElementById("pause");
+const nextButton = document.getElementById("forward");
+const autoNextsButton = document.getElementById("auto-forward");
+let imageSource = document.getElementById("pic");
+let images = [
+  "pictures/charles-postiaux-Q6UehpkBSnQ-unsplash.jpg",
+  "pictures/chris-karidis-nnzkZNYWHaU-unsplash.jpg",
+  "pictures/ferdinand-stohr-PeFk7fzxTdk-unsplash.jpg",
+  "pictures/joshua-ang-Gf_KqXHU-PY-unsplash.jpg",
+  "pictures/wael-hneini-QJKEa9n3yN8-unsplash.jpg",
+];
+let i = 0;
```

Comment:
```suggestion
let i = 0;
```
In this example, we could use `quoteIndex` instead of `i`

Reply:
Just to make it a bit clearer what this variable is used for throughout your application